api/src/main/java/io/minio/MinioClient.java

```diff
@@ -3648,9 +3649,8 @@ public void enableVersioning(EnableVersioningArgs args)
 
     Map<String, String> queryParamMap = new HashMap<>();
     queryParamMap.put("versioning", "");
-    String config =
-        "<VersioningConfiguration xmlns=\"http://s3.amazonaws.com/doc/2006-03-01/\">"
-            + "<Status>Enabled</Status></VersioningConfiguration>";
+    VersioningConfiguration config = new VersioningConfiguration("Enabled");
```

Comment:
Its better to pass a `boolean` rather a string.

Reply:
done